### ISSUE TITLE
[ BUG ] | Low level of accessibility (73%), it should be improved (Lighthouse report). Prepare a PWA.

### ISSUE THREAD

**Issue by kruchai** [OlegScout/olegkruchay-vue#82]
**Describe the bug**
The accessibility level should be increased. 

See the lighthouse report.
https://builder-dot-lighthouse-ci.appspot.com/report.1617509004712.html

**Expected behavior**
Need a 💯 percent value for all the indexes.

**Smartphone:** 
Whole smartphones.
